TileStache/Goodies/VecTiles/transform.py

```diff
@@ -273,18 +273,16 @@ def road_sort_key(shape, properties, fid, zoom):
 
         # Explicit layer is clipped to [-5, 5] range
         layer = properties.get('layer')
-        if layer:
-            layer_float = to_float(layer)
-            if layer_float is not None:
-                layer_float = max(min(layer_float, 5), -5)
-                # The range of values from above is [5, 34]
-                # For positive layer values, we want the range to be:
-                # [34, 39]
-                if layer_float > 0:
-                    sort_val = int(layer_float + 34)
-                # For negative layer values, [0, 5]
-                elif layer_float < 0:
-                    sort_val = int(layer_float + 5)
+        if layer is not None:
+            layer = max(min(layer, 5), -5)
```

Comment:
Yup, I should have made a note, sorry. This change and mapzen/vector-datasource#285 are kind of interdependent.